py/examples/tour.py

```diff
@@ -220,35 +224,57 @@ def get_wave_completions(line, character, file_content):
         scripts=[ui.script(q.app.tour_assets + '/loader.min.js')],
         script=ui.inline_script(content=template, requires=['require'], targets=['monaco-editor']),
         layouts=[
-            ui.layout(breakpoint='xs', zones=[
+            ui.layout(
+                breakpoint='xs',
+                zones=[
+                    ui.zone('mobile_header'),
+                    ui.zone('main', direction=ui.ZoneDirection.COLUMN,
+                            zones=[
```

Comment:
No need for being explicit.

```suggestion
                    ui.zone('main', zones=[
```